sacro-app/src/start-server.js

```diff
@@ -21,12 +21,13 @@ const startServer = async () => {
 
   // Spawn the server process
   const p = findAppPath();
-  const serverProcess = spawn(p, {
-    env: {
-      SACRO_APP_TOKEN: RANDOM_SECRET,
-      PORT: freePort,
-    },
-  });
+  const env = {
+    SECRET_KEY: RANDOM_SECRET,
```

Comment:
Are we okay to reuse this RANDOM_SECRET here, when we're later adding it to the cookie?

Reply:
Thinking on it some more, I think this is probably fine.